pages/common/git-rebase.md

```diff
@@ -17,3 +17,7 @@
 - Abort a rebase in-progress:
 
 `git rebase --abort`
+
+- Rebase your local branch by specifying new base commit and old base commit:
+
+`git rebase --onto {{NEW-BASE-COMMIT}} {{OLD-BASE-COMMIT}}`
```

Comment:
Yes, we came up with a finalized style guide a bit later. You can fix those too in another PR if you would like to :smiley: